src/ontology/uberon-edit.obo

```diff
@@ -224667,6 +224685,73 @@ is_a: UBERON:0002553 ! anatomical cavity
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-07-05T11:30:55Z xsd:dateTime
 
+[Term]
+id: UBERON:8480023
+name: skin of lateral lumbar region of abdomen
+def: "A zone of skin that is part of the lateral lumbar region of abdomen." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of flank" EXACT []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0000180 ! lateral lumbar region of abdomen
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:11:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8480024
+name: skin of sacral region
+def: "A zone of skin that is part of the sacral region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0005473 ! sacral region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:14:27Z xsd:dateTime
+
+[Term]
+id: UBERON:8480025
+name: skin of clavicle region
+def: "A zone of skin that is above the clavicle zone in the pectoral girdle region." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0001421 ! pectoral girdle region
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:17:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8480026
+name: skin of iliac crest region
+def: "A zone of skin that is above the iliac crest in the pelvic girdle bone/zone." [https://orcid.org/0000-0002-0819-0473]
+synonym: "skin of posterior iliac crest region" BROAD []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0007830 ! pelvic girdle bone/zone
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-08-29T14:19:16Z xsd:dateTime
+
+[Term]
+id: UBERON:8480027
+name: temple
+def: "The region where the frontal, parietal, temporal, and sphenoid bones fuse." [WikipediaVersioned:Temple_(anatomy)&oldid=1104547137]
+comment: This term does not refer to the "gena" of insects, which temple in that instance is a synonym (listed here: http://portal.hymao.org/projects/32/public/ontology_class/show_expanded/482).
```

Comment:
instead of listed here, I think make this a dbxref

Reply:
@shawntanzk I do not understand where to put this xref. Because we are saying that temple defined here in UBERON:8480027 does not refer to the temple refered in the xref mentioned in the comment